docs/users/jsc.rst

```diff
@@ -3,64 +3,128 @@
 Jülich Supercomputing Centre (JSC) - Germany
 ============================================
 
-:website: http://www.fz-juelich.de/ias/jsc
+:website: http://www.fz-juelich.de/jsc
 
-:institute: Forschungszentrum Jülich - http://www.fz-juelich.de/portal/EN
+:institution: Forschungszentrum Jülich - http://www.fz-juelich.de/portal/EN
 
-:contact: *(to be completed)*
+:contact: Damian Alvarez (d.alvarez@fz-juelich.de)
```

Comment:
maybe having multiple contacts here makes sense in your case? include Alan/Markus?